src/components/AddItem.js

```diff
@@ -84,35 +135,55 @@ const AddItem = () => {
           onChange={handleChange}
           required
         />
-        <fieldset>
+        <fieldset className={classes.fieldset}>
           <legend>How soon will you buy this again?</legend>
           <input
             type="radio"
             name="frequency"
             id="soon"
             value={7}
             onChange={handleChange}
+            className={classes.radio}
             required
           ></input>
-          <label htmlFor="soon">Soon</label>
+          <label
+            className={`${classes.soon} ${classes.radioLabel}`}
+            htmlFor="soon"
+          >
+            Soon
+          </label>
           <input
             type="radio"
             name="frequency"
             id="kind-of-soon"
             value={14}
             onChange={handleChange}
+            className={classes.radio}
             required
           ></input>
-          <label htmlFor="kind-of-soon"> Kind of soon</label>
+          <label
+            className={`${classes.kindOfSoon} ${classes.radioLabel}`}
+            htmlFor="kind-of-soon"
+          >
+            {' '}
+            Kind of soon
+          </label>
```

Comment:
I _do_ have a question about this snippet here because I'm just curious... What does the `{' '}` do?

Reply:
To be honest, I'm not sure, I think my editor must have just automatically added it 🤣

Reply:
Haha yeah that happens to me too 😆